app/cdap/components/AddTransformation/CircularProgressBar/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Typography } from '@material-ui/core';
+import React, { useState } from 'react';
+import { useStyles } from './styles';
+
+const MatchMeter: React.FC<{ value: number }> = ({ value }) => {
+  const classes = useStyles();
+
+  const getInlineStyles = (): string => {
+    if (value < 100) {
+      return '#E97567';
+    } else {
+      return '#8BCC74';
+    }
+  };
+  return (
+    <>
+      <Typography component="div" className={classes.progress}>
+        <Typography component="div" className={classes.barOverflow}>
+          <Typography
+            style={{
+              borderBottomColor: getInlineStyles(),
```

Comment:
move all css to styles.ts or use styled components if possible

Reply:
Done